include/dlaf/blas/tile_extensions.h

```diff
@@ -39,16 +42,12 @@ using matrix::Tile;
 template <Backend B, class T, Device D>
```

Comment:
Do we need `@brief` on the first line of the docstring above? Edit: actually, is it needed in general? I just saw it on the tile blas overloads which is what made me comment, but it's not used on most functions/classes. Is it only required to have the `@brief` description show up on the other overloads?

Reply:
We use the syntax where `@brief` is not needed:
```
/// Brief doc
///
/// Full doc
```